VideoSyncClass/VideoSync.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from subprocess import run, call
+from pathlib import Path
+import yaml
+import os
+from scipy.io import wavfile
+from scipy.io.wavfile import read as wavread
+import mne
```

Comment:
sort imports. Two tools are `isort` and `ruff`, and there are IDE plugins for both. Most projects I interact with are using ruff nowadays (which also does lots of other linting tasks, not just import sorting).